Hunting Queries/SecurityEvent/ADAccountLockouts.yaml

```diff
@@ -0,0 +1,20 @@
+id: e7642e6e-cf27-46ec-a4b9-e4475228fead
+name: AD Account Lockout
+description: |
+  'Detects Active Directory account lockouts'
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - InitialAccess
+  - CredentialAccess
+relevantTechniques:
+  - T1531
+query: |
+  let timeframe = 7d;
+  SecurityEvent
+  | where TimeGenerated >= ago(timeframe)
+  | where EventID == 4740
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), count() by Account
```

Comment:
We are transitioning to just use StartTime/EndTime versus StartTimeUtc/EndTimeUtc labeling.  UTC is not always how the UX is represented depending on what the User has chosen for their settings in the UX.  Also, we prefer to label items like count so that we don't get the default "count_" label, but not a major issue.  Lastly, it is best to build out context in the results such as what Domain controller this occurred on, what host the logon attempt came from and so on.  Would you be willing to include that?  Documentation on the event is here - https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4740

Reply:
hi shain, thank you for additional comments ;) i will definitely update the query as requested. just give me some time ;)